### ISSUE TITLE
[1822] Error on bid pass

### ISSUE THREAD

**Issue by JaySixA** [tobymao/18xx#9603]
https://18xx.games/game/134278 has an issue.  When I hit "Pass (Bids)", it says easterly1 passes, then gives an "auto actions do not match" error and resets to my turn.

**Comment by michaeljb** [comment]
Does the issue persist if you refresh the page before hitting pass again?

**Comment by JaySixA** [comment]
Yes, I also tried putting in my bids in a different order.  I think the problem is with the auto-pass because of the wording of the error, not with my bids, and I'm no expert.

**Comment by JaySixA** [comment]
On my game, it went to easterly1 without error.  I'm not sure if they had an autopass or that was skipped, or if it's functioning correctly now.
